ipam/allocator_test.go

```diff
@@ -532,7 +532,7 @@ func TestAllocatorFuzz(t *testing.T) {
 		addressIndex := rand.Int31n(int32(subnet.Size()))
 		alloc := allocs[allocIndex]
 		addr := address.Add(subnet.Addr, address.Offset(addressIndex))
-		err := alloc.Claim(name, address.MakeCIDR(subnet, addr), true, true)
+		err := alloc.Claim(name, address.MakeCIDR(subnet, addr), true, true, returnFalse)
```

Comment:
I've just pushed a refactor in daa369b which makes all the changes in this file superfluous, and instead only a single line needs changing.